<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPMainWindow</class>
 <widget class="QMainWindow" name="MPMainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Minuet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QFrame" name="loadingIndicator">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #FFC;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Loading....</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QFrame" name="currentSongInfo">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #DEF;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="1">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="6">
           <widget class="Line" name="line_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="7">
           <widget class="QLabel" name="songYear">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Year</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLabel" name="songGenre">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Genre</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="songAlbum">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Album name</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="songArtist">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Artist Name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="songTitle">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Song Title</string>
            </property>
           </widget>
          </item>
          <item row="2" column="7">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Now Playing:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="songArt">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/icons/default_cover.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="playControl">
         <item>
          <widget class="QLabel" name="seekLabel">
           <property name="text">
            <string>0:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MPFancySlider" name="seekSlider">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Seek current song.</string>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="seekDuration">
           <property name="text">
            <string>0:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="volumeSlider">
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Adjust volume</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="notchTarget">
            <double>5.000000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="volumeLabel">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>100%</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="MPPlaylistTableView" name="playlistView">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMusicPlayer">
    <property name="title">
     <string>Minuet</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuMedia">
    <property name="title">
     <string>Media</string>
    </property>
    <addaction name="actionPlay_Pause"/>
    <addaction name="actionStop_2"/>
    <addaction name="actionNext_2"/>
    <addaction name="actionPrevious_2"/>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove_from_Playlist"/>
   </widget>
   <addaction name="menuMusicPlayer"/>
   <addaction name="menuMedia"/>
   <addaction name="menuPlaylist"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddToPlaylist"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionStop"/>
   <addaction name="actionPausePlay"/>
  </widget>
  <action name="actionAdd">
   <property name="text">
    <string>Add to Playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAddToPlaylist">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add to playlist.</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/previous.png</normaloff>:/icons/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous song in the playlist.</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop the current song.</string>
   </property>
  </action>
  <action name="actionPausePlay">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play / Pause the current song.</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next song in the playlist.</string>
   </property>
  </action>
  <action name="actionPlay_Pause">
   <property name="text">
    <string>Play / Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionNext_2">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPrevious_2">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionStop_2">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected song from playlist.</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionRemove_from_Playlist">
   <property name="text">
    <string>Remove from Playlist</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Minuet</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MPFancySlider</class>
   <extends>QSlider</extends>
   <header>mpfancyslider.h</header>
  </customwidget>
  <customwidget>
   <class>MPPlaylistTableView</class>
   <extends>QTableView</extends>
   <header>mpplaylisttableview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MPMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlay_Pause</sender>
   <signal>triggered(bool)</signal>
   <receiver>actionPausePlay</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNext_2</sender>
   <signal>triggered()</signal>
   <receiver>actionNext</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrevious_2</sender>
   <signal>triggered(bool)</signal>
   <receiver>actionPrevious</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop_2</sender>
   <signal>triggered()</signal>
   <receiver>actionStop</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mainToolBar</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>mainToolBar</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_from_Playlist</sender>
   <signal>triggered()</signal>
   <receiver>actionRemove</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
